network/wsNetwork.go

```diff
@@ -1821,6 +1830,18 @@ func (wn *WebsocketNetwork) sendPeerConnectionsTelemetryStatus() {
 			InstanceName:         peer.InstanceName,
 			DuplicateFilterCount: peer.duplicateFilterCount,
 		}
+		// unwrap websocket.Conn, requestTrackedConnection, rejectingLimitListenerConn
+		var uconn net.Conn = peer.conn.UnderlyingConn()
+		for {
```

Comment:
Is there an expected upper bound on this? If so, will be safer if this loop is bounded to say 10x that expected bound, just to avoid the infinite loop situation.

Reply:
Agree with Shant and JJ, need to bound here